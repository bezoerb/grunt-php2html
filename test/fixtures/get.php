GET:
<?php
/**
 * Copyright (c) 2014 Ben ZÃ¶rb
 * Licensed under the MIT license.
 * http://bezoerb.mit-license.org/
 */

print_r($_GET);