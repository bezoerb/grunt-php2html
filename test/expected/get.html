GET: Array
(
    [test] => 42
    [arr] => Array
        (
            [0] => 1
            [1] => 2
            [2] => 3
            [3] => 4
        )

    [obj] => Array
        (
            [a] => 1
            [b] => 2
            [c] => 3
        )

)
